src/schema/rules/sidecars/mri.yaml

```diff
@@ -187,13 +187,19 @@ MRITimingParameters:
           should only be used when the volume timing is critical for interpretation
           of the data, such as in ASL or variable echo time fMRI sequences.
     InversionTime: recommended
+    SliceEncodingDirection: recommended
```

Comment:
Presumably SliceEncodingDirection is not defined for 3D acquisitions?

Reply:
Good point